.travis.yml

```diff
@@ -45,10 +50,16 @@ before_install:
   - echo "MAVEN_OPTS='-Xmx2048m -XX:MaxPermSize=512m'" > ~/.mavenrc
   - if [ "$TRAVIS_OS_NAME" == "osx" ]; then export JAVA_HOME=$(/usr/libexec/java_home); fi
   - if [ "$TRAVIS_OS_NAME" == "linux" ]; then jdk_switcher use "$CUSTOM_JDK"; fi
+  # python sdk environment settings
+  - export TOX_ENV=py27
+  - if [ "$TRAVIS_OS_NAME" == "osx" ]; then export TOX_HOME=$HOME/Library/Python/2.7/bin; fi
+  - if [ "$TRAVIS_OS_NAME" == "linux" ]; then export TOX_HOME=$HOME/.local/bin; fi
 
 install:
   - travis_retry mvn -B install clean -U -DskipTests=true
+  - travis_retry pip install tox --user `whoami`
 
 script:
+  - travis_retry $TOX_HOME/tox -e $TOX_ENV
```

Comment:
It runs for about ~210 seconds for 398 existing tests. It can be a separate a row in the matrix however Python SDK also benefits from testing on osx and linux. What do you prefer?